RELEASING.md

```diff
@@ -12,7 +12,20 @@ types.
 In the future we might be able to update the build to work
 without having JDK 8 installed, by using the `-release` option.
 
-### MinGW
+### GPG and sbt-pgp setup
+ 
+* Check the [sbt-pgp usage](https://www.scala-sbt.org/sbt-pgp/usage.html) for any setup steps you may still need, for example:
+```
+sbt> set pgpReadOnly := false
+sbt> pgp-cmd gen-key
```

Comment:
The release script has this done from the command line: https://github.com/akka/akka/blob/master/project/scripts/release#L15

Reply:
Yes but it's buried there, I think it is helpful to call out more upfront in setup.

Reply:
We could also just create a new GPG key for each release in the script to demonstrate the absurdity of the current security story. Would anyone even notice?

Reply:
I think I should re-word this to be clear that it is a one-time.

Reply:
Sorry, that was not completely serious...

The point is that the only reason for signing is to associate an artifact with a well-known-good actor. A new key will never be well-known, so it's somewhat paradoxical that we explicitly mention that you need to create a key before making a release. On the other hand, it's also quite ironical that we could just recreate a key for every release and no one would notice because no one will actually check that artifacts were signed by a well-known and well-meaning person (or even check the signature at all).

Reply:
NP I know ;) but what I meant here is that it is part of a one-time initial setup, vs per release.